tests/spec/fs.appendFile.spec.js

```diff
@@ -110,6 +110,19 @@ describe('fs.appendFile', function() {
     });
   });
 
+  it('should error when the symbolic link does not exist', function(done) {
+    var fs = util.fs();
+    var contents = 'This is a file.';
+     
+    fs.appendFile('/newfile', contents, { encoding: 'utf8' }, function(error) {  
+    expect(error).to.exist;
+    expect(error.code).to.equal('ENOENT');
+    expect(result).not.to.exist;
```

Comment:
In my opinion, ENOENT means the file or directory is not exist, so you do not need to write  120 line.

Reply:
Thank you for your input, I did not need the line at 120, and have changed actually ENOENT to EINVAL to pass this test.